packages/edit-site/src/components/sidebar-dataviews/default-views.js

```diff
@@ -54,6 +54,36 @@ export const DEFAULT_VIEWS = {
 			icon: pages,
 			view: DEFAULT_PAGE_BASE,
 		},
+		{
+			title: __( 'Published' ),
+			slug: 'published',
+			icon: pages,
```

Comment:
@jameskoster in lieu of a better option, I reused the icon used for "All pages" here.